lib/ansible/modules/network/aci/aci_tenant.py

```diff
@@ -20,161 +20,217 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
-
 module: aci_tenant
-short_description: Direct access to the APIC API
+short_description: Manage tenants on Cisco ACI fabrics
 description:
-    - Offers direct access to the APIC API
-author: Cisco
-extends_documentation_fragment: aci
+- Manage tenants on a Cisco ACI fabric.
```

Comment:
This description no doubt could be improved with additional information.
We could also add `notes:` for some additional stuff the user needs to be aware of.